shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -121,12 +124,18 @@ void TizenViewElementary::RegisterEventHandlers() {
   evas_object_callbacks_[EVAS_CALLBACK_RESIZE] =
       [](void* data, Evas* evas, Evas_Object* object, void* event_info) {
         auto* self = reinterpret_cast<TizenViewElementary*>(data);
-        if (self->view_) {
+        if (self->view_delegate_) {
           if (self->container_ == object) {
-            int32_t width = 0, height = 0;
-            evas_object_geometry_get(object, nullptr, nullptr, &width, &height);
+            int32_t x = 0, y = 0, width = 0, height = 0;
+            evas_object_geometry_get(self->container_, &x, &y, &width, &height);
+
+            evas_object_size_hint_min_set(self->container_, width, height);
+            evas_object_size_hint_max_set(self->container_, width, height);
+
+            EvasObjectResizeWithMinMaxHint(self->image_, width, height);
```

Comment:
Directly accessing `self->container_` and setting value within this callback and calling `EvasObjectResizeWithMinMaxHint` goes against the delegation you talked about so far.

If the container and image need to resize, call `sefl->view_delegate_->Resize(??)` and handle the appropriate process inside.